connectors/v2/rbmaradio.js

```diff
@@ -34,14 +34,14 @@ Connector.getDuration = function () {
 	var $progressControl = $('.c-player__controls > .c-player__progress > progress');
 	var duration = '';
 	if ($progressControl) {
-		duration = $progressControl.attr('max'); 
+		duration = $progressControl.attr('max');
```

Comment:
There's missing `parseInt` call at this line. The `getDuration` function should return numbers.